vpr/src/draw/ui_setup.h

```diff
@@ -61,6 +61,13 @@ void search_setup(ezgl::application* app);
  */
 void routing_button_setup(ezgl::application* app);
 
+/*
```

Comment:
Needs /** to be doxygen.
Delete the duplicate comment in the .cpp file (it's the same as this one).